File: Python/1116.py
n = int(input())

for i in range(1, n + 1):
    x, y = map(int,input().split())
    
    if y == 0:
        print('divisao impossivel')
        
    if y != 0:
        divisao = x / y
        print('{:.1f}'.format(divisao))

File: CSharp/1116.cs
using System;
using System.Linq;
using System.Globalization;


class URI {

    static void Main(string[] args) 
    {
        CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
        int[] x;
        int n = int.Parse(Console.ReadLine());
        for(int i =0; i<n; i++)
        {
            x=Console.ReadLine().Split().Select(int.Parse).ToArray();

            if(x[1]==0)
            {Console.WriteLine("divisao impossivel");}
            else{Console.WriteLine($"{(decimal)x[0]/x[1]:F1}");}
        }
    }

}

